sktime/datasets/tests/test_uschange.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3 -u
+# coding: utf-8
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["Sebastiaan Koel"]
+__all__ = []
+
+import pandas as pd
+from sktime.datasets import load_uschange
+
+
+def test_uschange():
+    X, y = load_uschange()
+
+    X_test = pd.read_pickle('uschange_X.pkl')
```

Comment:
The pickle files are the X and y in pickle files and then it just compares these with the dataframe/serie returned by the function. Now the entire dataset is in it but for larger datasets I would've done i.e. the first 20 rows. I discussed this with Omar and he preferred not to have binary files, but just assert the column names, shapes and dtypes.  I think it's better this way because it checks the actual data as well, if someone accidentally changed the csv. But I don't mind changing it to what Omar suggested. Do you want me to change it?

Reply:
Ah okay, I think it's enough to have checks for what @big-o suggested, I don't want to have two copies of the data, the data itself is not that important I think, it's just toy data. Also someone may accidentally change the copy of the data, right? So a better test would be to download and compare the dataset from a reference source but I think this is not necessary. If you want to write unit tests for the loading function, it would be very useful to generalise them to test the other loading functions too (only the output object types, data types).

Reply:
Okay just removed the pickle files and only use assertions. Also tried to make a general dataset testing function which can be used for other datasets as well.